### PR TITLE
Add doc on cluster configuration

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
- Adds doc on how to modify terraform to change Plural defaults

### CODE REVIEW

File: pages/operations/cluster-configuration.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+---
+title: Cluster Configuration
+description: DevOps workflows that involve editing your cluster Terraform.
```
Comment (swoodward90) [inline pages/operations/cluster-configuration.md:3]: nit: "cluster's Terraform"

Comment (swoodward90) [reply pages/operations/cluster-configuration.md:3]: ?

**Review by michaeljguarino — APPROVED** [review]

File: pages/operations/cluster-configuration.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+---
+
+Plural offers a set of sane defaults to spin up a one-size-fits-all Kubernetes cluster, but there will be cases
```
Comment (swoodward90) [inline pages/operations/cluster-configuration.md:6]: this is a really good paragraph. that's all.

File: pages/operations/cluster-configuration.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        value  = "SPOT"
+        effect = "NO_SCHEDULE"
+      },
+    ],
+}
+```
+{% /tab %}
+
+{% tab title="Azure" %}
+Currently azure has an annoying chicken-egg issue with the requirement at least one node pool is created.  Terraform manages this poorly by forcing cluster recreation if the default node pool changes.  To ensure no instability, we strongly recommend you confirm any node topology changes do not interfere with the default node pool on the AKS cluster.
```
Comment (swoodward90) [inline pages/operations/cluster-configuration.md:107]: nit: Azure

Comment (swoodward90) [reply pages/operations/cluster-configuration.md:107]: "the requirement **that** at least..."

File: pages/operations/cost-management.md [hunk 4]
```diff
@@ -33,6 +33,7 @@ As with any operations that involve taking down resources, you will likely encou
 If you start deploying a lot of applications and resources, you may start seeing cost go up faster than expected due to the fixed cost of installing Kubernetes and its services on every node.
```
Comment (swoodward90) [inline pages/operations/cost-management.md:33]: nit: kill first "start"

File: pages/operations/cluster-configuration.md (hunk 2 above)
Comment (avaidyanatha) [reply pages/operations/cluster-configuration.md:6]: aw thanks :)

File: pages/operations/cluster-configuration.md (hunk 1 above)
Comment (avaidyanatha) [reply pages/operations/cluster-configuration.md]: fixed, I like cluster's more anyway but either is technically fine

File: pages/operations/cluster-configuration.md (hunk 3 above)
Comment (avaidyanatha) [reply pages/operations/cluster-configuration.md]: done

File: pages/operations/cost-management.md (hunk 4 above)
Comment (avaidyanatha) [reply pages/operations/cost-management.md]: done

**Pull request merged**